api_by_features/battery.json

```diff
@@ -0,0 +1,45 @@
+{
+    "Baterry": {
+    	"description": "Ability to read level and mode of charge of baterry.",
+        "browser": {
+        	"API": "Baterry Status API",
+        	"spec_url": "http://www.w3.org/TR/battery-status/",
+        	"working group": ""
```

Comment:
Hi Dom,      Thanks for reminding us. We will work on this as soon as possible. Best,Leandro.                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone.                                                                                                                                                                                                                From: Dominique Hazael-MassieuxSent: segunda-feira, 20 de abril de 2015 12:16To: w3c-webmob/web-api-gapReply To: w3c-webmob/web-api-gapSubject: Re: [web-api-gap] Remove battery.md, add battery.json and add gitignore (#13)In api_by_features/battery.json:
> @@ -0,0 +1,45 @@
> +{
> +    "Baterry": {
> +    	"description": "Ability to read level and mode of charge of baterry.",
> +        "browser": {
> +        	"API": "Baterry Status API",
> +        	"spec_url": "http://www.w3.org/TR/battery-status/",
> +        	"working group": ""

as I alluded to the other day during the Skype call, I think we should use the same structure for "browser" and for the other platforms, i.e. "name" instead of "API", and use "url" instead of "spec_url" (and change "docs" to "url" in the other platforms section). We should also avoid having "working group" in there since that's data we can get from elsewhere.

—Reply to this email directly or view it on GitHub.